### DISCUSSION TITLE
When does github support multiple languages

### DISCUSSION THREAD

**Discussion by GMYXDS** [community/community#58627] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

When does github support multiple languages?

**Comment by the-officialjosh** [comment]
It is a good feature, no?

**Comment by AlbinXXX** [comment]
GitHub is like a polyglot prodigy, supporting multiple programming languages for developers to showcase their skills. 
However, it doesn't provide language translation or support for spoken languages. GitHub focuses on code, leaving spoken languages to human communication. You can still communicate in your preferred spoken language within the GitHub community, but remember, GitHub's superpower lies in fostering collaboration through programming languages. Let your code do the talking and save the linguistic marvels for human interactions. Happy coding, polyglot superheroes! (345 characters)
